practices/risk_analysis.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: risk-analysis
+stages:
+  - development
+short_description: Risk Analysis and Management is a key project management practice to ensure that the least number of surprises occur while the project is underway
+tags:
+  - risk analysis
+  - software documentation
+keywords:
+  - risk analysis
+  - documentation
+  - task
+---
+
+# Risk Analysis
+
+**TL;DR**
+
+Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks.
```

Comment:
_Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks._ from https://searchsecurity.techtarget.com/definition/risk-analysis